apps/api-gateway/src/issuance/dtos/issuance.dto.ts

```diff
@@ -1,18 +1,36 @@
-import { IsArray, IsNotEmpty, IsOptional, IsString, IsEmail, ArrayMaxSize } from 'class-validator';
+import { IsArray, IsNotEmpty, IsOptional, IsString, IsEmail, ArrayMaxSize, ValidateNested } from 'class-validator';
 import { ApiProperty } from '@nestjs/swagger';
 import { Transform, Type } from 'class-transformer';
 import { toNumber } from '@credebl/common/cast.helper';
 
-interface CredentialOffer {
```

Comment:
Could you please tell me why this is removed?

Reply:
I removed the interface and created a DTO for it because I have to apply class validators to it.